src/components/ProjectList/listOfProjects.js

```diff
@@ -332,7 +332,13 @@ const projectList = [
     description: 'Production-Grade Container Scheduling and Management',
     tags: ['Go', 'Container', 'Orchestration'],
   },
-
+  {
+    name: 'styled-system',
+    imageSrc: 'https://camo.githubusercontent.com/8b3dc6438530a7240e952b187e40bd8380090f64/68747470733a2f2f7374796c65642d73797374656d2e636f6d2f6c6f676f2e737667',
+    githubLink: 'https://github.com/styled-system/styled-system',
```

Comment:
Can you please update the link to `https://github.com/styled-system/styled-system/contribute`

Reply:
@sara-02  updated.